cosmos/providers/databricks/notebook.py

```diff
@@ -31,6 +31,13 @@ class DatabricksNotebookOperator(BaseOperator):
                 databricks_conn_id="databricks_conn",
                 job_clusters=job_cluster_spec,
                 notebook_params=[],
+                notebook_packages=[
+                    {
+                        "pypi": {
+                            "package": "simplejson"
+                        }
+                    },
+                ]
```

Comment:
I think we need to be careful in the documentation that we make it clear that anything added at the taskgroup level will be added to all notebooks. This is especially important since this is a functionality that doesn't exist within databricks. It might make sense to not have this example at the notebook documentation and instead put it in the taskgroup, but I'm open on this.

Reply:
Removed the docstring note from notebook and it already exists in the taskgroup workflow.py docstring.

Reply:
Also, updated the docs to mention the suggestion.